operatorcourier/const.py

```diff
@@ -0,0 +1,97 @@
+general_required_fields = [
```

Comment:
Some of these fields are checked for in the default operator-courier validation. There may be some overlap in these checks and the one's already present.

Reply:
Yeah I realize that but I'm not sure how to bypass that in a way that the default validation and ui validation stay in sync. I'm open to suggestions but IMO a little overlap could be better than more complexity